docs/release-notes/release-notes-0.1.0.md

```diff
@@ -0,0 +1,310 @@
+not enter the contract goes in a change address. This is no different from when
+you make a regular transaction, however because the input UTXOs are locked in
+advance of broadcasting the actual transaction that spends them, you will see
+the amount locked in the wallet until the swap actually takes place.
+
+Depending on the asset, there may be a wallet setting on the Wallets page to
+pre-size funding UTXOs to avoid this over-locking, but (1) it involves an extra
+transaction that pays to yourself before placing the order, which has on-chain
+transaction fees that may be undesirable on chains like BTC, and (2) it is only
+applied for limit orders with standing time-in-force since the the UTXOs are
+only locked until the swap transaction is broadcasted, which is relatively brief
+for taker-only orders that are never booked.
+
+### Epochs
+
+An important concept with DCRDEX is that newly submitted orders are processed in
+short windows of time called **epochs**, the length of which is part of the
+server's market configuration, but is typically on the order of 10 seconds. When
+a valid order is received by the server, it enters into the pool of epoch orders
+before it is matched and/or booked. The motivation for this approach is
+described in detail in the DCRDEX specification. The Your Orders table will show
+the status of such orders as "epoch" until they are matched at the end of the
+epoch, as described in the next section.
+
+Order cancellation requests are also processed in the epoch with trade
+(market/limit) orders since a cancellation is actually a type of order. However,
+from the user's perspective, cancelling an order is simply a matter clicking the
+cancel icon for one of their booked orders.
+
+### Matching
+
+When the end of an epoch is reached, the orders it includes are then matched
+with the orders that are already on the book. A key concept of DCRDEX order
+matching is a deterministic algorithm for shuffling the epoch orders so that it
+is difficult for a user to game the system. To perform the shuffling of the
+closed epoch prior to matching, clients with orders in the epoch must provide to
+the server a special value for each of their orders called a **preimage**, which
+must correspond to another value that was provided when the order was initially
+submitted called the **commitment**. This is done automatically by dexc,
+requiring no action from the user.
+
+If an order fails to match with another order, it will become either **booked**
+or **executed** with no part of the order filled. The Your Orders table displays
+the current status and remaining quantity of each of a user's orders. If an
+order does match with another trade order, the order status will become
+**settling**, and atomic swap negotiation begins.
+
+### Settlement
+
+When maker orders (on the book) are matched with taker orders from an epoch, the
+atomic swap sequence begins. No action is required from either user during the
+process.
```

Comment:
maker/taker is random if both epoch?

Reply:
Yes, user's don't know for sure if they are maker or taker until matching. So one of the epoch orders gets booked first becoming the maker, to which the second one matches as taker.

I suppose the current wording here implies that all epoch orders are takers?  Perhaps parenthesizing "from an epoch" will make this clearer.